export-arduino-lib.py

```diff
@@ -0,0 +1,77 @@
+import os
+import sys
+import zipfile
+import tempfile
+
+
+OUTPUT_NAME = "libsmp.zip"
+INCLUDED_DIRS = ["docs"]  # directories copied as-is with their tree
+EXTRACTED_DIRS = ["include", "src"]  # directories from which we copy all the files and put them at the root
+INCLUDED_FILES = ["COPYING", "README.md", "library.properties"]  # files in the root that we include
+EXCLUDED_FILES = [
+    ".gitignore",
+    "libsmp-private-posix.h",
+    "serial-device-avr.c",
+    "serial-device-posix.c",
+    "serial-device-win32.c"
+]  # files to be excluded globally
+CONFIGURATION_PARAMETERS = {
+    "config.h": [],
+    "libsmp-config.h": [
+        ("SMP_MESSAGE_MAX_VALUES", 8, "The maximum number of arguments in a message"),
+        ("SMP_SERIAL_FRAME_MAX_FRAME_SIZE", 128, "The maximum frame size in bytes")
+    ]
+}
+
+if len(sys.argv) > 1:
+    OUTPUT_NAME = sys.argv[1]  # If provided, we use the filename given by the user
```

Comment:
By doing so, we can end up with bad things, ie:
```bash
$ ./export-arduino-lib.py -h
```
will result in an archive named `-h`.

I think we should either use `argparse` or do like for parameters: prompt user